airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/manifest.yaml

```diff
@@ -421,7 +421,7 @@ definitions:
     cursor_granularity: "PT1H"
     step: P1D
     cursor_datetime_formats:
-      - "%Y-%m-%d %H:%M:%S"
+      - "%Y-%m-%d"
       - "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
As a future action, perhaps we should have this as standard behavior in the CDK?

I think these are all ISO8601 / RFC3339 compliant, although we generally prefer "T" as the delimiter, space is also a common delimiter for RFC3339. While we probably should prefer a specific format, I can't see a reason not to parse all formats and standardize on _always_ writing a specific and opinionated one.